From 369c5ea6fd113cd8de7a50917bc5795833755924 Mon Sep 17 00:00:00 2001
From: himta ram <hram@codeaurora.org>
Date: Wed, 29 Jan 2020 11:49:19 +0530
Subject: [PATCH 09/10] Bluetooth: Support to know if there is CS call

Add the isCsCallInProgress API to know the CS call
in progress.
Add the isCsCallInProgress API instead of non-protected
system broadcast with action codeaurora.telecom.action.CALL_TYPE.

CRs-Fixed: 2676955
Change-Id: Idaf514948d4a6a6be5f72e98d67dbae56a153e6e
---
 binder/android/bluetooth/IBluetoothHeadsetPhone.aidl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl b/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
index 9a51d1d43..04f834186 100644
--- a/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
+++ b/binder/android/bluetooth/IBluetoothHeadsetPhone.aidl
@@ -32,6 +32,7 @@ interface IBluetoothHeadsetPhone {
   boolean listCurrentCalls();
   boolean queryPhoneState();
   boolean isHighDefCallInProgress();
+  boolean isCsCallInProgress();
 
 
   // Internal for phone app to call
-- 
2.28.0

